TestCases/sliding_interface/uniform_flow/uniform_SM.cfg

```diff
@@ -0,0 +1,267 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
Yep.
By the way I have other reg test that I run locally on my desktop, I didn't put all of those to avoid making checks too long (I remember we had some issues with timeout limits in the past).
The crucial ones are there already.
If timeout is no longer a problem,  I can include the 5 zone pipe in place of the single stage (2zone only) so to have a more demanding test on multi-zone fluid problems.